bundle/manifests/node-healthcheck-operator-manager-role-aggregator_rbac.authorization.k8s.io_v1_clusterrole.yaml

```diff
@@ -0,0 +1,10 @@
+aggregationRule:
```

Comment:
something looks off here.
this shouldn't it be under something? e.g. spec/annotations etc.

Reply:
the keys of the yaml get sorted, so you see it on top. but its valid cluster role

Reply:
https://kubernetes.io/docs/reference/access-authn-authz/rbac/#aggregated-clusterroles